docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+    randbot:
+        build: .
+        env_file: .env
+        environment:
+          - REDIS_URL=redis://redis:6379
```

Comment:
Should probably lock this up with a password at some point

Reply:
Yeah, we definitely should. The other thing we could do is remove the ports key under the `redis` service and it'll only be accessible from inside the docker network so it won't even have external access. 

The only reason I say this is I'm not sure how well `hubot-brain` which is the redis interaction layer in `hubot` works with an authenticated Redis. I mean it should but I'm not making any assumptions.